app/components/course/corse.component.ts

```diff
@@ -82,5 +82,16 @@ import { Component, Input } from '@angular/core';
 })
 
 export class CourseComponent {
-    @Input() course:any
-}
\ No newline at end of file
+    @Input() private course: { id: string, title: string,
```

Comment:
Extract interface definition and use that type identifier instead of extended object notation. https://www.typescriptlang.org/docs/handbook/interfaces.html

Reply:
done